js/autotrack/common.ts

```diff
@@ -108,11 +148,16 @@ const getFiberNodeComponentHierarchyTraversal = currNode => {
 // * hierarchy - the string hierarchy representation of 'hierarchyArray'.
 // * heapIgnoreProps - the final set of HeapIgnore props that applies to the target component. Used
 //     to determine whether to include target text and/or ignore the entire interaction.
-const getHierarchyStringFromTraversal = hierarchyArray => {
-  let currentHeapIgnoreProps = _.mapValues(BASE_HEAP_IGNORE_PROPS, () => false);
+const getHierarchyStringFromTraversal: (
+  hierarchyArray: ComponentHierarchyTraversalElement[]
+) => HierarchyResult = hierarchyArray => {
+  let currentHeapIgnoreProps: HeapIgnoreProps = _.mapValues(
+    BASE_HEAP_IGNORE_PROPS,
+    () => false
+  );
 
   // Map each hierarchy element to its string representation, considering HeapIgnore specs.
-  const hierarchyStrings = hierarchyArray
+  const hierarchyStrings: string = hierarchyArray
```

Comment:
Should this be plural?  At the end we get a single string out (of concatenated strings, but it's still just a single string out the other side...) . Perhaps just `hierarchyString`?

Reply:
Yep, fixed